modules/cluster-logging-forwarding-separate-indices.adoc

```diff
@@ -0,0 +1,69 @@
+// Module is included in the following assemblies:
+//cluster-logging-external
+:_content-type: PROCEDURE
+[id="cluster-logging-forwarding-separate-indices_{context}"]
+= Forwarding JSON logs from containers in the same pod to separate indices
+
```

Comment:
For the abstract to appear in search results, it must be the first paragraph below the title.